rllib/agents/dqn/apex.py

```diff
@@ -63,10 +63,21 @@
         "num_gpus": 1,
         "num_workers": 32,
 
-        "buffer_size": 2000000,
         # TODO(jungong) : add proper replay_buffer_config after
         #     DistributedReplayBuffer type is supported.
-        "no_local_replay_buffer": True,
+        "replay_buffer_config": {
```

Comment:
Let's make these old keys == DEPRECATED_VALUE.

Then in validate_config:
```
if config["buffer_size"] != DEPRECATED_VALUE:
    deprecation_warning(..., error=False)
    config["replay_buffer_config"]["capacity"] = config["buffer_size"]
```

Do the above if-checks for all deprecated keys to make sure existing users' configs don't fully break.

Reply:
I currently do this in the trainer's _create_local_replay_buffer_if_necessary function and am moving this as per your request. 💯